docs/samples/contact-center/index.html

```diff
@@ -85,19 +85,47 @@ <h2 class="collapsible">
   </section>
   </div>
 
-  <div class="box">
-    <section class="section-box">
-      <h2 class="collapsible">
-        Agent Desktop Using Webex CC SDK
-        <i class="arrow fa fa-angle-down" aria-hidden="true"></i>
-      </h2>
-
-      <div class="section-content">
+    <div class="box">
+      <section class="section-box">
+        <h2 class="collapsible">
+          Agent Desktop Using Webex CC SDK
```

Comment:
Let's update this title to say AgentLogin or StationLogin

Reply:
addressed